buildSrc/src/main/java/com/uber/okbuck/core/model/android/AndroidAppTarget.java

```diff
@@ -179,6 +178,22 @@ private Keystore extractKeystore() {
     return null;
   }
 
+  @Nullable
+  private String getExoManifest() {
+    String mainManifest = getMainManifest();
+
+    if (mainManifest != null) {
+      String exoManifest = mainManifest.replace("main", "exo");
```

Comment:
What's the purpose of the string replace. This is very specific and only works if folks name their exo variant 'exo'. Can we not hardcore the names here?